app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -62,6 +68,11 @@ public static Double strToDouble(String value) {
         return Utils.isEmpty(value) ? null : Double.parseDouble(value);
     }
 
+    @SuppressWarnings("PMD")
+    public static int strToInt(String value) {
+        return Utils.isEmpty(value) ? 1 : Integer.parseInt(value);
```

Comment:
Why 1 if it is empty?

Reply:
Actually the problem is i cannot return null for primitives so i have to return some integer value, so what value should i return??

Reply:
Why is this needed in the first place, it is done automatically